docs/user_guide/discretisation/EqualFrequencyDiscretiser.rst

```diff
@@ -5,122 +5,320 @@
 EqualFrequencyDiscretiser
 =========================
 
-The :class:`EqualFrequencyDiscretiser()` sorts continuous numerical variables into
-contiguous equal frequency intervals, that is, intervals that contain approximately the
-same proportion of observations. The limits of the intervals are calculated according
-to percentiles or quantiles utilising `pandas.qcut()`. You decide the number of
-intervals.
+The :class:`EqualFrequencyDiscretiser()` is a discretization method that applies the pandas.qcut() function to divide continuous numerical variables into equal-frequency bins. These bins contain roughly the same number of observations, with boundaries set at specific quantile values determined by the desired number of bins (q parameter). This method ensures a uniform distribution of data points across the range of values, enhancing the handling of skewed data and outliers.
```

Comment:
This is nit: sometimes it is used discretization and some time discretisation. Could not be possible to make it uniform above all the package :-) ?

Reply:
Done! Using discretization instead. Thanks :)